### ISSUE TITLE
onBlur validation on check box group is triggered early

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#237]
### Description of the bug

When using a keyboard to navigate the form the checkbox group will activate while leaving the first checkbox. This will give the user a premature warning that they have to fill out elements they have not reached yet. This is also a problem for screen reader users who will get an error message that will interrupt them while navigating within the check box group. 

~~The validation should be triggered when leaving the group~~

NB: This is not an issue with radio buttons where you use the arrow keys to choose between the different radios. The validation is triggered when tabbing to the next component. 

### Steps To Reproduce

1. Go to a form with a group of checkboxes
2. Try using the keyboard no navigate
3. while tabbing from the first checkbox to the second you should trigger a validation message

**Comment by olemartinorg** [comment]
I suspect this problem has been mitigated by the solution in #98 - required fields will no longer give you premature warnings when navigating through the form using the keyboard. Maybe we can close this as a duplicate, @Febakke?

**Comment by olemartinorg** [comment]
Correction: It looks to have been partially mitigated, but the issue still remains for radio groups and checkboxes. I also noticed that, when navigating the form with the keyboard, it is very easy to change the value of a radio selection. Holding the keyboard button down changes the input as fast as possible. The way saving the form is triggered, this creates a constant stream of network requests to POST new data - we should consider debouncing these events for radio/checkboxes just the way we did it for inputs.
